content/english/about/_index.md

```diff
@@ -2,32 +2,29 @@
+Your friends and neighbors across Oakland who came together after the murder of George Floyd to try to make changes in policing at the city level. We met after listening to hours upon hours of Oakland government meetings and [live-tweeting under #oakmtg](https://twitter.com/search?q=%23oakmtg). We rooted each other on as we tried to champion the #defundOPD movement alongside hundreds of community voices.
 
 Then we became infuriated and fascinated by the lack of transparency and lies we saw so blatantly in front of us. We became hooked on understanding what was happening and why our city is so disorganized. We want to distill our ongoing work to understand our government into something that our friends and neighbors can understand. We especially want to uplift the voices of those in the Black community, which is the community most impacted by police violence and systemic racism. We may not even vote the same way amongst ourselves, and we think that’s okay, as long as we are all informed.
 
-### 
-
 ### What we do:
 
 We've created our first [voter guide](/guide) for the November 2020 election to try to increase transparency around the candidates based on what we've learned from these city meetings.
 
-We also live tweet Oakland meetings using the "#oakmtg" hashtag. Come join us! We started out with and are actively engaged in "last mile" civic participation. We call in to various meetings, including Oakland City Council and Oakland Police Commission meetings, during the public comment sections. This can and does cause Council Members and Commissioners to change what is being voted on.
+We also [live-tweet Oakland meetings using the "#oakmtg" hashtag](https://twitter.com/search?q=%23oakmtg). Come join us! We started out with and are actively engaged in "last mile" civic participation. We call in to various meetings, including Oakland City Council and Oakland Police Commission meetings, during the public comment sections. This can and does cause Council Members and Commissioners to change what is being voted on.
 
 We want to do more.
 
-This site was made for us to be more effective at "last mile" civic engagement and enable more folks to join us. There's a LOT to take in, so we made this website to make it easier to keep track of what's going on. We will build this site out over time, particularly after the election, as we keep track of what promises were made by which politicians.
+This site was made for us to be more effective at "last mile" civic engagement and enable more folks to join us. There's a LOT to take in, so we made this website to make it easier to keep track of what's going on. We will build this site out over time, particularly after the election, as we track promises made by electeds and appointees.
 
-If you'd like to join us, please shoot us an email at XXX@XXX.com.
+If you'd like to join us, please shoot us an email at info@oaktmg.club.
```

Comment:
We can also use `oakmtg@gmail.com` if you prefer

Reply:
Should update to the new Google group.